src/main/java/chimp/core/Ui.java

```diff
@@ -0,0 +1,101 @@
+package chimp.core;
+import java.util.HashMap;
+
+import chimp.task.Task;
+
+public class Ui {
+    private static final String DIVIDER = "____________________________________________________________\n";
+
+    /**
+     * Retrieves a HashMap of phrases used by the Chimp application.
+     *
+     * @return A HashMap containing phrases for various actions.
+     */
+    HashMap<String, String> getPhrases() {
+        HashMap<String, String> phrases = new HashMap<>();
+        String greet = " Hello! I'm Chimp\n" +
+                " What can I do for you?\n";
+        String bye = "Bye. Hope to see you again soon!\n";
+        String mark = "Nice! I've marked this task as done: \n";
+        String unmark = "OK, I've marked this task as not done yet: \n";
+        String hoo = "HOO-HOO-HOO-HOO, I don't know what that means\n";
+        String delete = "Noted. I've removed this task:\n";
+        String find = "Here are the matching tasks in your list:\n";
+
+        phrases.put("greet", greet);
+        phrases.put("bye", bye);
+        phrases.put("mark", mark);
+        phrases.put("unmark", unmark);
+        phrases.put("hoo", hoo);
+        phrases.put("delete", delete);
+        phrases.put("find", find);
+
+        return phrases;
+    }
```

Comment:
Perhaps it would be best to store a `HashMap<String, String> phrases` instance as a static field since you would be referring to it whenever you call the `say` method?